### PR TITLE
Feature/cu 86bzpb1cx/users microfrontend app

### PR DESCRIPTION
**Pull request by marvinIsSacul**

### CODE REVIEW

File: packages/header-app/src/components/openhim.appbar.component.tsx [hunk 1]
```diff
@@ -91,6 +91,15 @@ const pages: Page[] = [
   {name: 'CHANNELS', link: '#!/channels'},
   {name: 'ROLES', link: '#!/roles'},
```
Comment (drizzentic) [inline packages/header-app/src/components/openhim.appbar.component.tsx:92]: We need to remove this and move it under users as per design

File: packages/header-app/src/components/openhim.appbar.component.tsx [hunk 2]
```diff
@@ -91,6 +91,15 @@ const pages: Page[] = [
...
   {name: 'CLIENTS', link: '#!/clients'},
+  {
+    name: 'USERS',
+    children: [
+      {name: 'Manage Users', link: '#!/users'}
+      // {name: 'TASKS', link: '#!/tasks'},
+      // {name: 'VISUALIZER', link: '#!/visualizer'},
+      // {name: 'CONTACT LISTS', link: '#!/groups'},
```
Comment (drizzentic) [inline packages/header-app/src/components/openhim.appbar.component.tsx:100]: Replace the commented code with the actual items as per design.

File: packages/header-app/src/components/openhim.appbar.component.tsx [hunk 3]
```diff
@@ -117,15 +126,27 @@ export default function OpenhimAppBar() {
   const [isAdmin, setIsAdmin] = useState(false)
   const [anchorElNav, setAnchorElNav] = useState<null | HTMLElement>(null)
   const [anchorElUser, setAnchorElUser] = useState<null | HTMLElement>(null)
```
Comment (drizzentic) [inline packages/header-app/src/components/openhim.appbar.component.tsx:128]: Is this constant different from `const [anchorElUsers, setAnchorElUsers]`

File: packages/root-config/src/microfrontend-layout.html [hunk 4]
```diff
@@ -54,6 +52,11 @@
           <application name="@jembi/dashboard-app"></application>
         </div>
       </route>
+      <route path="#!/users">
+        <div style="flex-grow: 1">
+          <application name="@jembi/users-app"></application>
+        </div>
+      </route>
     </div>
     <application name="@jembi/footer-app"></application>
```
Comment (drizzentic) [inline packages/root-config/src/microfrontend-layout.html:61]: The footer should come below all other apps. The portal apps are inject using the `openhim-console-ems-container` id

File: packages/users-app/src/types/index.ts [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  'certificates-manage': boolean
+  'logs-view': boolean
+  'import-export': boolean
+}
+
+export type Role = {
+  _id: string
+  name: string
+  permissions: Permission
+}
```
Comment (drizzentic) [inline packages/users-app/src/types/index.ts:45]: Do we need this here?

File: packages/users-app/src/screens/list.users.screen.tsx [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+  const users = data || []
+
+  React.useEffect(() => {
+    refetch()
+  }, [])
+
+  // React.useEffect(() => {
+  //   // refetch()
+  // }, [search])
+
```
Comment (drizzentic) [inline packages/users-app/src/screens/list.users.screen.tsx:51]: remove unused code

File: packages/users-app/src/components/helpers/permission.chip.component.tsx [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+export const PermissionChip = (props: PermissionChipProps) => {
+  return (
+    <Stack sx={{display: 'flex'}} direction="row" spacing={1}>
+      {Array.isArray(props.data) &&
+        props.data.map(channel => <Chip key={channel} label={channel} />)}
+    </Stack>
+  )
+}
```
Comment (drizzentic) [inline packages/users-app/src/components/helpers/permission.chip.component.tsx:15]: Do we need to use permissions when managing users?

File: packages/header-app/src/components/openhim.appbar.component.tsx (hunk 3 above)
Comment (marvinIsSacul) [reply packages/header-app/src/components/openhim.appbar.component.tsx:128]: Yes. Each dropdown menu item needs to have its own anchor element state.

**Review by drizzentic — APPROVED** [review]
LGTM!

**Pull request merged**